website/docs/r/vpc.html.markdown

```diff
@@ -102,6 +103,7 @@ In addition to all arguments above, the following attributes are exported:
 * `default_security_group_id` - The ID of the security group created by default on VPC creation
 * `default_route_table_id` - The ID of the route table created by default on VPC creation
 * `ipv6_association_id` - The association ID for the IPv6 CIDR block.
+* ipv6_cidr_block_network_border_group - The Network Border Group Zone name
```

Comment:
all attribute names in the docs should be in single backtick `. 

please mention if an attribute has a required attribute in order to be set (aka: `assign_generated_ipv6_cidr_block`)